docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,160 @@ weight = -17
 
-In the meantime if your want to know more about what we have been up to and what is coming sign up to our online GrafanaCon conference.
+This major release of Grafana contains a lot of new features and enhancements including the next step in our Observability story, powerful new features for manipulating, transforming and doing math on data and the first version of Usage Analytics in Grafana Enterprise.
+
+The Grafana 7.0 stable will be released on the 18th of May. In the meantime if your want to know more about what we have been up to and what is coming sign up to our online GrafanaCon conference.
 
 [{{< figure src="/assets/img/blog/GrafanaCONline.jpg" max-width="800px" lightbox="false" caption="GrafanaCONline May 13-29" >}}](https://grafana.com/about/events/grafanacon/2020/?source=blog)
+
+The main highlights are:
+
+- **New Panel Editor** Redesign based on community feedback
+- **Explore:** New tracing UI and support for visualizing Jaeger and Zipkin traces.
+- **Enterprise** Usage Analytics and Presence
+- **Transformations** Transformations and simple Math operations for all data sources.
+- **Field overrides** Automatically configure panels with data from queries.
+- **Table** New Table panel.
+- **Plugins** New plugins platform
+- **Tutorials** New tutorials section
+- **Cloudwatch** Cloudwatch Logs
+- **Breaking change** PhantomJS removed
+
+## New Panel Editor
+
+In Grafana 7 we have redesigned the UI for editing panels. The first visible change is that we have separated panel display settings to a right hand side pane that you can collapse or expand depending on what your focus is on. With this change we are also introducing our new unified option model & UI for defining data configuration and display options. This unified data configuration system powers a consistent UI for setting data options across visualizations as well as making all data display settings data driven and overridable.
+
+## New tracing UI
+
+With Grafana 7.0 we are adding major support for distributed tracing. This adds an important telemetry mode to complement the existing support for metrics and logs. Traces allow you to follow how single requests travelled through a distributed system. We are starting with an integrated trace viewer and two new built-in data sources: Jaeger and Zipkin.
+
+You can use the new trace view in Explore either directly and search for a particular trace or you can configure Loki to detect trace IDs in the log lines and link directly to a trace timeline pulled from Jaeger or Zipkin data source.
+
+In the future we will add more workflows and integrations so that correlating between metrics, logs and traces is even easier.
+
+## Transformations
+
+With Grafana 7.0, data transformations are now a core part of the data processing that happens before the visualization step. All users can now do simple transformations no matter what data source or visualization type they use.
```

Comment:
```suggestion
With Grafana 7.0, data transformations are now a core part of the data processing that happens before the visualization step. All users can now do simple transformations no matter what data source or visualization type they use.
```
Not just visualizing data from anywhere, in Grafana 7 you can transform it too. By chaining a simple set of point and click transformations users will be able join, pivot, filter, re-name and calculate to get the results they need. Perfect for operations across queries or data sources missing essential data transformations.